library/ssl_tls13_client.c

```diff
@@ -757,6 +811,25 @@ int mbedtls_ssl_tls13_write_identities_of_pre_shared_key_ext(
             break;
         }
     }
+    else
+#if defined(MBEDTLS_SSL_SESSION_TICKETS)
+    if( psk_type == MBEDTLS_SSL_TLS1_3_PSK_RESUMPTION )
+    {
+#if defined(MBEDTLS_HAVE_TIME)
+        mbedtls_time_t now = mbedtls_time( NULL );
+        obfuscated_ticket_age =
+            (uint32_t)( now - ssl->session_negotiate->ticket_received ) +
```

Comment:
```suggestion
           ( (uint32_t)( now - ssl->session_negotiate->ticket_received ) * 1000 ) +
```
`ssl_tickets.c` assumes that `mbedtls_time()` returns a value in seconds (as time() does in Linux and Windows). Let's be consistent with this here and thus multiply the time difference by 1000 as the ticket age is defined to be in ms by the TLS 1.3 spec.